migrations/008_add_doc_chunks.sql

```
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS doc_chunks (
+    id         SERIAL PRIMARY KEY,
+    source     TEXT NOT NULL,         -- e.g. "docs-axiom/AXIOM-SPEC.md"
+    heading    TEXT,                  -- nearest H2/H3 heading above this chunk
+    content    TEXT NOT NULL,         -- chunk text (~800 chars)
+    embedding  vector(1536),
+    indexed_at TIMESTAMPTZ DEFAULT NOW()
+);
+
+CREATE INDEX IF NOT EXISTS doc_chunks_embedding_idx
+    ON doc_chunks USING ivfflat (embedding vector_cosine_ops)
+    WITH (lists = 10);
```

SQLFluff (4.1.0)

[error] Line 13, Position 1: Found unparsable section: 'CREATE INDEX IF NOT EXISTS doc_chunks_em...'

(PRS)